### ISSUE TITLE
Allow multiple urls in `*_scan` functions

### ISSUE THREAD

**Issue by scsmithr** [GlareDB/glaredb#1225]
## Summary

Eg

```
select * from parquet_scan(['./path/1.parquet', './path/2.parquet'])
```

**Comment by scsmithr** [comment]
My initial thoughts here were we'd push it down to the exec and assume that all files provided are in the same bucket / use the same object store. If they differ, we could just error as I would think wanting to scan multiple parquet files from different buckets would be an exceptional use case.

The logical union is interesting, and definitely could be something we could implement to support the multiple buckets use case. However I think as a first pass, we just assume everything is in the same bucket for now.

**Comment by scsmithr** [comment]
This was closed with #1314
